Extension/src/common.ts

```diff
@@ -342,6 +342,13 @@ export function resolveVariables(input: string | undefined, additionalEnvironmen
         return "";
     }
 
+    // jsonc parser may assign a non-string object to a string.
+    if (typeof(input) !== "string") {
+        const inputAny: any = input;
+        input = inputAny.toString();
```

Comment:
Have you tried this with an explicit `undefined`?  You might need to check the input for null before calling toString().

Reply:
line 341 should guard against `undefined`

Reply:
Thanks, that was outside of the diff.